### PR TITLE
[DASH-2371] Fix innertoolbar when hosting have long name

### PR DESCRIPTION
**Pull request by powerczuk**
Scripts view:

screen shot 2016-11-21 at 15 27 57

Templates view:

screen shot 2016-11-21 at 15 27 43

Data Endpoint Preview:

screen shot 2016-11-17 at 16 49 32

Data Objects view:

screen shot 2016-11-17 at 16 47 13

Hosting Files view:

screen shot 2016-11-17 at 16 35 15

### CODE REVIEW

File: src/apps/Hosting/Files.js [hunk 1]
```diff
@@ -81,10 +90,22 @@ const HostingFilesView = React.createClass({
...
+    return hostingDetails && !isLoading ? `Hosting: ${hostingDetails.name} (id: ${hostingDetails.id})` : '';
+  },
+
+  getTruncatedTitle() {
+    const { hostingDetails, isLoading } = this.state;
+
+    if (hostingDetails && !isLoading) {
+      const hostingName = `Hosting: ${hostingDetails.name}`;
+
+      return `${_.truncate(hostingName, { length: 55 })} (id: ${hostingDetails.id})`;
```
Comment (patrykkopycinski) [inline src/apps/Hosting/Files.js]: it should be done in css!

File: src/apps/Hosting/Files.js [hunk 2]
```diff
@@ -242,25 +263,33 @@ const HostingFilesView = React.createClass({
...
           backButtonTooltip="Go Back to Hosting"
         >
+          <Tooltip
+            label={wholeTitle}
+            horizontalPosition="right"
+            style={styles.toolbarTooltip}
+          >
+            <span style={styles.customToolbarTitle}>{truncatedTitle}</span>
+          </Tooltip>
+          <div style={{ flex: 1 }} />
```
Comment (patrykkopycinski) [inline src/apps/Hosting/Files.js]: extract styles to `const styles`

File: src/apps/Hosting/Files.js [hunk 3]
```diff
@@ -242,25 +263,33 @@ const HostingFilesView = React.createClass({
     const currentInstance = SessionStore.getInstance();
     const currentInstanceName = currentInstance && currentInstance.name;
     const hostingUrl = this.getHostingUrl();
-    const pageTitle = this.getToolbarTitle();
+    const wholeTitle = this.getWholeTitle();
```
Comment (patrykkopycinski) [inline src/apps/Hosting/Files.js]: `fullTitle`

**Comment by gh3r** [comment]
LGTM

**Pull request merged**